.travis.yml

```diff
@@ -1,13 +1,48 @@
 language: node_js
 node_js:
   - "10"
-
+services:
+  - docker
 cache:
+  yarn: true
   directories:
-    - "node_modules"
+    - node_modules
+# install
+  # nothing
+
+jobs:
+  include:
+    - stage: Test and Build
+      script:
+      - cd styleguide && yarn install --production=false --frozen-lockfile --non-interactive --ignore-engines
+      - yarn run styleguide:build
+      - yarn install --frozen-lockfile --non-interactive --ignore-engines
+      # - yarn run ci
+    - script:
+      - docker build --build-arg BUILD_COMMIT=$TRAVIS_COMMIT -t humanconnection/webapp .
```

Comment:
` -t humanconnection/webapp` are you sure? Shouldn't it be rather ` -t humanconnection/nitro-webapp`? Shall we start uploading images to `webapp` after we drop the legacy code?

Reply:
Would be fine for me, but before we need to push it manually

Reply:
`webapp` is there but not used by any of our services. I'm fine to use `webapp`.